Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -171,10 +171,10 @@ class BrowserLocationView: UIView {
     }
 
     private func updateTextWithURL() {
-        if let httplessURL = url?.absoluteStringWithoutHTTPScheme(), let baseDomain = url?.baseDomain() {
+        if let protocollessURL = url?.absoluteStringWithoutProtocol(), let baseDomain = url?.baseDomain() {
```

Comment:
No. That bug is for the top sites cell, which usually shows the title but falls back to the URL. Top Sites now groups by domain, and has limited space, so we’d like to show just the important bits of the domain and path.

The URL bar should always show the actual page you’re viewing.